cmd/duffle/relocate.go

```diff
@@ -4,15 +4,22 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 
-	"github.com/pivotal/image-relocation/pkg/image"
-	"github.com/pivotal/image-relocation/pkg/pathmapping"
-	"github.com/pivotal/image-relocation/pkg/registry"
+	"github.com/deislabs/duffle/pkg/imagestore/builder"
+
```

Comment:
nit: Could you please group the imports from Duffle?

Reply:
Fixed.